roles/redis/templates/redis.conf.j2

```diff
@@ -898,6 +898,7 @@ replica-announce-port {{ redis_port }}
 # user itential on allkeys allchannels allcommands -asking -cluster -readonly -readwrite -bgrewriteaof -bgsave -failover -flushall -flushdb -psync -replconf -replicaof -save -shutdown -sync >{{ redis_user_itential_password }}
 # user repluser on allchannels +psync +replconf +ping >{{ redis_user_repluser_password }}
 # user sentineluser on allchannels +multi +slaveof +ping +exec +subscribe +config|rewrite +role +publish +info +client|setname +client|kill +script|kill >{{ redis_user_sentineluser_password }}
+# user prometheus on -@all +@connection +memory -readonly +strlen +config|get +xinfo +pfcount -quit +zcard +type +xlen -readwrite -command +client -wait +scard +llen +hlen +get +eval +slowlog +cluster|info -hello -echo +info +latency +scan -reset -auth -asking >{{ redis_user_prometheus_password }}
```

Comment:
Where did you get these permissions from?

Reply:
I got those from a redis_exporter git repo (not sure if it is *the* repo).

https://github.com/oliver006/redis_exporter

Reply:
Yeah, it looks like that's the one being used in the role.

Reply:
The readme lists two acl, one for redis and one for sentinel.  So maybe we need to add some more permissions when sentinel is enabled.

Reply:
Yeah, this is going to be a problem.  Since the redis roles are separated, I'm not sure how to enable the correct permissions.  I tried adding the sentinel permissions when clustering is off and the service won't start.

Reply:
Alternatively, we could add the most inclusive list of permissions (even if we don't need them because sentinel might not be running).

Reply:
We will need the sentinel acls for Frontier.

Reply:
> Alternatively, we could add the most inclusive list of permissions (even if we don't need them because sentinel might not be running).

Redis didn't like it when I gave the prometheus user the sentinel permissions when sentinel wasn't running.